cmd/run.go

```diff
@@ -155,6 +155,8 @@ the data document with the following syntax:
 	runCommand.Flags().VarP(authorization, "authorization", "", "set authorization scheme")
 	runCommand.Flags().VarP(logLevel, "log-level", "l", "set log level")
 	runCommand.Flags().VarP(logFormat, "log-format", "", "set log format")
+	runCommand.Flags().StringArrayVar(&params.ConfigOverrides, "set", []string{}, "override config values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
+	runCommand.Flags().StringArrayVar(&params.ConfigOverrideFiles, "set-file", []string{}, "override config values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2)")
```

Comment:
Nit: `override config values with files on the command line` and same comment as above for the parenthetical?